### PR TITLE
Add flow config and environment __DEV__

### PR DESCRIPTION
**Pull request by matuzalemsteles**
## Use of flow
I am testing the use of flow on Fiber and seems to be very promising, to know more information of him access [here](https://flow.org/en/). What do you think of this?

- [ ] `isomorphic/FiberComponent`
- [ ] `isomorphic/FiberCreateComponent`
- [ ] `isomorphic/FiberElement`
- [X] `isomorphic/FiberRender`
- [ ] `isomorphic/FiberUpdateQueue`
- [ ] `renderers/dom/FiberDOMEvents`
- [ ] `renderers/dom/FiberDOMRender`

To check flow errors, run `yarn run flow`.

## Use of `__DEV__`
I added a configuration so that we can use the `__DEV__` variable to display information in a block, such as error messages or warnings. Things we do not want in a production environment.

**Example**:
```javascript
if (__DEV__) {
    console.log('In development');
}
```

Blocks that use `__DEV__` are removed when compiled for production.

### CODE REVIEW

File: src/isomorphic/render.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+
 import { buildComponentFromVNode } from './createComponent'
 
 let component;
 
+type Vnode = {
+  nodeName: string | ?void,
+  attributes: Array<any> | void,
+  children: string | Array<any> | void
+};
```
Comment (clovisdasilvaneto) [inline src/isomorphic/render.js]: Hey @matuzalemsteles , 

What do you think in create a separated folder, like flow-types and add a file, named as vnode.js, with the Vnode declarations?

Comment (matuzalemsteles) [reply src/isomorphic/render.js]: Sounds great, I liked it.

Comment (clovisdasilvaneto) [reply src/isomorphic/render.js]: o/

**Comment by clovisdasilvaneto** [comment]
Hey @matuzalemsteles can you rebase and send the PR again?

**Pull request closed without merging**